extensions/smallrye-graphql/deployment/src/test/java/io/quarkus/smallrye/graphql/deployment/GraphQLTest.java

```diff
@@ -193,6 +250,7 @@ public void testContext() {
     private static Map<String, String> configuration() {
         Map<String, String> m = new HashMap<>();
         m.put("quarkus.smallrye-graphql.events.enabled", "true");
+        m.put("quarkus.native.add-all-charsets", "true");
```

Comment:
If I'm not mistaken unit tests don't support native mode, this would be only needed inside `integration-tests`